components/compositing/compositor.rs

```diff
@@ -1957,17 +1957,15 @@ impl<Window: WindowMethods> IOCompositor<Window> {
         let clip_rect_for_children = if masks_to_bounds {
             Rect::new(Point2D::zero(), clipped_layer_bounds.size)
```

Comment:
I'm not sure whether this case also needs to be changed... or why it is separate from the `else` case below it.